src/carplay.cs

```diff
@@ -1136,6 +1136,7 @@ interface CPTemplateApplicationScene
 
 		[iOS (13, 0)]
 		[Field ("CPTemplateApplicationSceneSessionRoleApplication")]
+		[Advice ("Use `UIWindowSceneSessionRole.CarTemplateApplication` instead.")]
 		NSString SessionRoleApplication { get; }
```

Comment:
it does not look help to keep around...

```csharp
#if !XAMCORE_4_0
		[Field ("CPTemplateApplicationSceneSessionRoleApplication")]
 		[Obsolete ("Use `UIWindowSceneSessionRole.CarTemplateApplication` instead.")]
 		NSString SessionRoleApplication { get; }
#endif
```

and the `[iOS (13,0)]` is not needed since it's already on the type itself.